### PR TITLE
Update communication.md

### PR DESCRIPTION
**Pull request by VolodymyrYakovenko**
- [ ] PR's title includes the challenge's name and language
- [x] PR has short and clear description of the challenge
- [ ] PR has appropriate labels and milestones for easy identification
- [ ] PR it is assigned to the owner
- [ ] reviewers are assigned
- [ ] the PR contributes only one focused change
- [x] the branch is up to date with `main`/`master`
- [ ] the code works when pulled and run locally
- [ ] all conflicts are resolved (if any)
- [ ] It is linked to an issue in the appropriate column of the project board
- [ ] feedback is addressed (if any, and if it is appropriate feedback.)

## README Documentation

- [ ] The solution is documented in `/src/README.md`
- [x] The markdown source is formatted
- [x] Spelling and grammar is correct in all text
- [ ] The markdown looks correct when you preview the file
- [ ] All links and images work
- [ ] The README documents the solution's behavior, strategy(ies) and implementation(s)
- [ ] There are use cases in code blocks to illustrate the function's behavior

## Python Files

- [ ] There is a module header
- [ ] There is a module docstring
- [ ] File names are in snake_case
- [ ] Test files are named `test_<module_name>.py`

## Function Docstring

- [ ] Behavior description
- [ ] Parameter description
- [ ] Return value description
- [ ] Include any assumptions
- [ ] Include 3 or more (passing!) doctests
- [ ] Include 1-2 use cases (if necessary)

## Function Implementation

- [ ] The solution is not is not copied from [djeada](https://github.com/djeada/Algorithms-And-Data-Structures)
- [ ] The code is auto-formatted
- [ ] The code has no (reasonable) linting mistakes
- [ ] Variables are named with snake_case
- [ ] The function has a clear and helpful name
- [ ] The file's name matches the function name
- [ ] The code follows the strategy as simply as possible
- [ ] Variable names are clear and helpful
- [ ] Comments explain the strategy (if necessary)
- [ ] There are type annotations
- [ ] (challenge) The code includes defensive assertions

## Unit Test Suite

- [ ] The test class has a helpful name in PascalCase
- [ ] The test class has a docstring
- Each unit test has
  - [ ] A helpful name
  - [ ] A clear docstring
  - [ ] Only one assertion
- [ ] All tests pass
- [ ] There are tests for defensive assertions
- [ ] There are tests for boundary cases
- [ ] Test suite includes black-box and glass-box tests

### CODE REVIEW

**Comment by dhossainali** [comment]
That's great Volodymyr. We also can add some points on communications, like channels through which our mainstream communications should be done, proper naming of the commits, and clear and concise description of what changes are made in a commit.

File: group_study/communication.md [hunk 1]
```diff
@@ -19,31 +27,33 @@
 
 | Day | How | The topic of discussion |
 | --- | :-: | ----------------------- |
-|     |     |                         |
+|  16.01.2024   |  https://meet.google.com/wfz-xkgb-wzs   |  Norms                       |
 
 ## Communication Channels
```
Comment (dhossainali) [inline group_study/communication.md:32]: I think under this section if we can add slack and WhatsApp as the channels of communication and providing some clarification of how and when we should use from a particular channel.

File: group_study/communication.md [hunk 2]
```diff
@@ -10,6 +10,14 @@
 # Communication
```
[the remark this answers is not in the document]
Comment (VolodymyrYakovenko) [reply group_study/communication.md:10]: The part that you underlined for me - it's a title of a section where the things, that you mentioned - are partially/fully descriebed.

File: group_study/communication.md (hunk 1 above)
Comment (VolodymyrYakovenko) [reply group_study/communication.md:32]: I will add a link to slack and that would be enough I guess. Describing every channel in slack... that would be a bit time consuming.

File: group_study/communication.md [hunk 3]
```diff
@@ -10,6 +10,14 @@
...
 
 ## Group Rules
```
[the remark this answers is not in the document]
Comment (VolodymyrYakovenko) [reply group_study/communication.md:12]: Will add this to the rules, thank you.
